frontend/packages/topology/src/components/page/LimitExceededState.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import {
+  Button,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+  Title,
+} from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { useActiveNamespace } from '@console/shared';
+
+type LimitExceededStateProps = {
+  onShowTopologyAnyway: () => void;
+};
+
+export const LimitExceededState: React.FC<LimitExceededStateProps> = ({ onShowTopologyAnyway }) => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const [namespace] = useActiveNamespace();
+  return (
+    <EmptyState>
+      <Title headingLevel="h4" size="lg">
+        {t(`topology~Limit Exceeded`)}
+      </Title>
+      <EmptyStateBody>
+        {t(
+          `topology~We're sorry, but we can't support large number of resources on topology at this time. Our bots are working relentlessly to improve the performance.`,
+        )}
+      </EmptyStateBody>
+      <Button variant="primary" onClick={() => history.push(`/search-page/ns/${namespace}`)}>
+        {t('topology~View resources on Search page')}
+      </Button>
```

Comment:
Use semantic HTML. Navigational links shouldn't be buttons that push to history. Use react-router with the button. See patternfly docs for example code.